misc/plunker-builder-templates/index.html

```diff
@@ -5,10 +5,10 @@
   <base href="." />
   <title>ng-bootstrap demo</title>
   <link rel="stylesheet" href="http://v4-alpha.getbootstrap.com/dist/css/bootstrap.min.css" />
-  <script src="https://unpkg.com/zone.js/dist/zone.js"></script>
-  <script src="https://unpkg.com/zone.js/dist/long-stack-trace-zone.js"></script>
+  <script src="https://unpkg.com/zone.js@%zoneJs%/dist/zone.js"></script>
```

Comment:
Maybe we could move content of this file into code so we could use ES6 template strings instead of inventing our own tempting syntax?